infra/modules/providers/azure/service-plan/variables.tf

```diff
@@ -19,18 +20,18 @@ variable "service_plan_name" {
 }
 
 variable "service_plan_tier" {
-    description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
-    default     = "Isolated"
+  description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
```

Comment:
We have some variables with type and some without. Can we include them as a default across all our variable definitions.